### PR TITLE
Add value property in some transactions

### LINKED ISSUE #187: Error when a market is created
Check the image below

### PR DESCRIPTION
**Pull request by mariano-aguero**
Closes #187 

#### Includes
- Add value property with '0x0' in some transactions, support for some wallets mobile like trust and pillar
- Added fix for division by zero error

### CODE REVIEW

File: app/src/services/market_maker.ts [hunk 1]
```diff
@@ -85,8 +94,12 @@ class MarketMakerService {
...
 
     const totalBalance = balanceOfForYes.add(balanceOfForNo)
-    const actualPriceForYes = divBN(balanceOfForNo, totalBalance)
-    const actualPriceForNo = divBN(balanceOfForYes, totalBalance)
+    const actualPriceForYes = !totalBalance.isZero()
+      ? divBN(balanceOfForNo, totalBalance)
+      : balanceOfForNo.toNumber()
+    const actualPriceForNo = !totalBalance.isZero()
+      ? divBN(balanceOfForYes, totalBalance)
+      : balanceOfForYes.toNumber()
```
Comment (mariano-aguero) [inline app/src/services/market_maker.ts]: @fvictorio Please take a look at this carefully

File: app/src/services/market_maker.ts [hunk 2]
```diff
@@ -66,7 +68,12 @@ class MarketMakerService {
     logger.log(`Add funding to market maker ${amount}`)
 
     try {
-      return this.contract.addFunding(amount, distributionHint)
+      const overrides = {
+        gasLimit: 750000,
```
Comment (fvictorio) [inline app/src/services/market_maker.ts]: Was this part of a test or do we need to hardcode the gas limit so high?

File: app/src/services/realitio.ts [hunk 3]
```diff
@@ -66,7 +66,7 @@ class RealitioService {
 
     // send the transaction and wait until it's mined
     const transactionObject = await this.contract.askQuestion(...args, {
-      value: ethers.utils.bigNumberify(value),
+      value: '0x0',
```
Comment (fvictorio) [inline app/src/services/realitio.ts]: This is already 0. We need to either remove the `value` parameter or to undo these changes. I'm fine with either option (actually, since this was already tested, maybe it's slightly better to just remove the parameter).

File: app/src/services/market_maker.ts (hunk 1 above)
Comment (fvictorio) [reply app/src/services/market_maker.ts]: The only way `totalBalance` can be zero is if both `balanceForYes` and `balanceForNo` are zero (or one of them is negative, which shouldn't happen). I believe this only happens if the market has no funding. This situation should probably be handled in some other way. But, as a quick fix, I think this fix is OK (maybe we could just use 0 instead of `balanceOfFor{Yes,No}.toNumber()` to make it less obscure).

File: app/src/services/market_maker.ts (hunk 2 above)
Comment (mariano-aguero) [reply app/src/services/market_maker.ts]: We need the gas limit high, also It throws an error

File: app/src/services/market_maker.ts (hunk 1 above)
Comment (mariano-aguero) [reply app/src/services/market_maker.ts]: I will use 0 instead of `balanceOfFor...`

File: app/src/services/realitio.ts (hunk 3 above)
Comment (mariano-aguero) [reply app/src/services/realitio.ts]: I will remove the parameter

File: app/src/services/market_maker.ts (hunk 2 above)
Comment (mariano-aguero) [reply app/src/services/market_maker.ts]: I remove the gas limit, and tested again with Trust Wallet and Pillar Wallet, and is working fine! So I think this was added to solve a problem with gnosis safe

File: app/src/services/market_maker.ts [hunk 4]
```diff
@@ -177,7 +189,18 @@ class MarketMakerService {
     const outcomeIndex = outcome === OutcomeSlot.Yes ? 0 : 1
     try {
       const outcomeTokensToSell = await this.contract.calcSellAmount(amount, outcomeIndex)
-      await this.contract.sell(amount, outcomeIndex, outcomeTokensToSell)
+
+      const overrides = {
+        gasLimit: 750000,
```
Comment (mariano-aguero) [inline app/src/services/market_maker.ts]: I will check this too
